assets/templates/partials/banners/census-tile-sm.tmpl

```diff
@@ -8,7 +8,6 @@
                     <img src="https://cdn.ons.gov.uk/assets/images/census-logo/logo-census-2021-white-landscape-cy.svg" alt="{{ localise "Census2021" .Language 1 }}" class="header__svg-logo"
                     xmlns="http://www.w3.org/2000/svg" focusable="false" width="209" height="40" viewBox="0 0 228 44">
                 {{ end }}
-                </img>
```

Comment:
here the same. you can probably add / to the <img {your stuff here} />

Reply:
As last comment